pinecone/core/client/api/hybrid_operations_api.py

```diff
@@ -0,0 +1,407 @@
+#
```

Comment:
@acatav @miararoy I'm not sure we should push this file to the client repo, which is exposed to customers.  
Since it's auto-generated code - we can always commit it later when we also wire it to the `Index` object.   

For now these files would just be sitting there (and installed to customers' machines who `pip install pinecone`) for no reason.